storage/blockdevice/source/SFDP.cpp

```diff
@@ -254,7 +268,7 @@ int sfdp_parse_sector_map_table(Callback<int(bd_addr_t, void *, bd_size_t)> sfdp
      * - sector map configuration detection commands
      * - configurations
      * - regions in each configuration
-     *  is variable -> the size of this table is variable
+     * are variable -> the size of this table is variable
      */
     auto smptbl_buff = std::make_unique<uint8_t[]>(sfdp_info.smptbl.size);
     if (!smptbl_buff) {
```

Comment:
This doesn't work, because `make_unique` throws `std::bad_alloc` instead of returning, but we use `-fno-exceptions`.